packages/react-heartwood-components/src/components/RecordSelectionList/RecordSelectionList.js

```diff
@@ -245,69 +248,78 @@ export default class RecordSelectionList extends Component<
 		const {
 			selectedIds,
 			unselectableIds,
-			renderRecord,
 			getRecordId,
 			canSelect,
 			canRemove,
 			onSelect,
-			onRemove
+			onRemove,
+			recordKeys
 		} = this.props
 		const { loadedRecords } = this.state
 		const record = loadedRecords[index]
-		const SelectionComponent = canSelect === 'one' ? Radio : Checkbox
 
 		return (
 			record && (
-				<div
+				<ListItem
 					className="record-selection__record-wrapper"
 					key={key}
 					style={{ ...style }}
-				>
-					{onSelect && canSelect && (
-						<SelectionComponent
-							className="record-selection__record-select"
-							onChange={() => {
-								onSelect(getRecordId(record), record)
-							}}
-							disabled={
-								unselectableIds &&
-								unselectableIds.indexOf(getRecordId(record)) >= 0
-							}
-							checked={
-								selectedIds && selectedIds.indexOf(getRecordId(record)) >= 0
-							}
-						/>
-					)}
-
-					<div className="record-selection__record-content" key={key}>
-						{renderRecord(record)}
-					</div>
-
-					{onRemove && canRemove && (
-						<Button
-							kind="simple"
-							className="record-selection__record-remove-btn"
-							disabled={false}
-							isSmall
-							icon={{ name: 'cancel_solid', className: 'btn__line-icon' }}
-							onClick={() => {
-								this.setState(
+					id={record.node[recordKeys.id]}
+					title={record.node[recordKeys.title]}
+					subtitle={record.node[recordKeys.subtitle]}
```

Comment:
@jtini records will not always be in the format of `{ node: { ... } }` so we'll need a more flexible way to access the record keys now that the record isn't rendered inline with the component. I'd suggest we use lodash `get` on these so that recordKeys don't have to map directly to a property of the record. e.g. `title={get(record, recordKeys.title, '')}` would allow setting the `title` recordKey to `title` or `someChild.title`